detail.scss

```diff
@@ -0,0 +1,51 @@
+.title {
+  width: 100%;
+  color: #000;
+  font-family: sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38px;
+  display:inline;
+  border-bottom:3px solid #BED23C;
+  border-width: 100%;
+}
+.genero {
+  width: 100%;
```

Comment:
Maybe this is better in the upper `div` 🤔

Reply:
Done